backend/test/routes/reports.test.js

```diff
@@ -20,12 +20,6 @@ testRoutes([
   //   capabilities: capability,
   //   type: routeTypes.General,
   // },
-  {
```

Comment:
are we not adding a test in its place?

Reply:
I would like to have a ticket to update how these route tests are performed, which should allow all of these tests to be reinstated (plus the other reports which never had tests). I will put something in the backlog for this.